packages/config/config/devices/0x015f/mh-3901z.json

```
@@ -0,0 +1,129 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH3901-Z",
+	"description": "Boiler Thermostat",
+	"devices": [
+		{
+			"productType": "0x3910",
+			"productId": "0x5102"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Temperature reporting",
```

```suggestion
			"label": "Temperature Reporting",
```
param labels should be **Title Case**

---

All labels are checked.